samanthas_telegram_bot/conversation/callbacks/student.py

```diff
@@ -0,0 +1,138 @@
+import logging
+
+from telegram import InlineKeyboardButton, InlineKeyboardMarkup, Update
+
+from samanthas_telegram_bot.api_queries import get_smalltalk_url, send_written_answers_get_level
+from samanthas_telegram_bot.conversation.auxil.assessment import prepare_assessment
+from samanthas_telegram_bot.conversation.auxil.callback_query_reply_sender import (
+    CallbackQueryReplySender as CQReplySender,
+)
+from samanthas_telegram_bot.conversation.auxil.message_sender import MessageSender
+from samanthas_telegram_bot.conversation.auxil.shortcuts import answer_callback_query_and_get_data
+from samanthas_telegram_bot.conversation.constants_enums import (
+    PHRASES,
+    CommonCallbackData,
+    ConversationState,
+)
+from samanthas_telegram_bot.conversation.custom_context_types import CUSTOM_CONTEXT_TYPES
+
+logger = logging.getLogger(__name__)
+
+
+async def store_communication_language_ask_non_teaching_help_or_start_review(
+    update: Update, context: CUSTOM_CONTEXT_TYPES
+) -> int:
+    """Stores communication language, asks about non-teaching help or starts review.
+
+    This callback is intended for students only.
+    If a student is 15 or older, asks about additional help. Otherwise, proceeds to review.
+    """
+    (
+        query,
+        context.user_data.communication_language_in_class,
+    ) = await answer_callback_query_and_get_data(update)
+
+    if context.user_data.student_age_from >= 15:
+        await CQReplySender.ask_non_teaching_help(context, query)
+        return (
+            ConversationState.NON_TEACHING_HELP_MENU_OR_PEER_HELP_FOR_TEACHER_OR_REVIEW_FOR_STUDENT
+        )
+
+    await MessageSender.ask_review(update, context)
+    return ConversationState.REVIEW_MENU_OR_ASK_FINAL_COMMENT
+
+
+async def ask_communication_language_or_start_assessment_depending_on_learning_experience(
+    update: Update, context: CUSTOM_CONTEXT_TYPES
+) -> int:
+    """Starts assessment or asks for additional help the student requires. No data is stored here.
+
+    * For students that have been learning English for 1 year or more, starts assessment.
+    * For students that have been learning English for less than 1 year, stores that they
+      need an oral interview (skipping the assessment). Then asks about communication language.
+    """
+    query, data = await answer_callback_query_and_get_data(update)
+
+    if data == "year_or_more":
```

Comment:
How about some constant?

Reply:
I know, but this string is only used here and then in the CSV file, so here the constant will only be for the sake of a constant. [Same in `common.py`](https://github.com/samanthasgroup/samanthas-telegram-bot/blob/students-bot-1st-version/samanthas_telegram_bot/conversation/callbacks/common.py#L237). And maybe the same for the multiple literals in `teachers.py`.

Reply:
Ok, got it